metals/src/main/scala/scala/meta/internal/metals/Embedded.scala

```diff
@@ -61,44 +60,53 @@ final class Embedded(
 }
 
 object Embedded {
-  def downloadSettings(
+  def fetchSettings(
       dependency: Dependency,
       scalaVersion: String
-  ): Settings =
-    new coursiersmall.Settings()
-      .withTtl(Some(Duration.Inf))
-      .withDependencies(List(dependency))
-      .addRepositories(
-        List(
-          coursiersmall.Repository.SonatypeReleases,
-          coursiersmall.Repository.SonatypeSnapshots
-        )
-      )
-      .withForceVersions(
+  ): Fetch = {
+    val repositories =
+      Repository.defaults().asScala ++
+        List(Repository.central(), Repository.ivy2Local())
```

Comment:
I don't see `SonatypeReleases` and `SonatypeSnapshots` being added now, which was done previously.

Reply:
Should be fine now, but could you double check the urls?

Reply:
The look right.